src/main/java/seedu/address/logic/commands/lesson/LessonFindCommand.java

```diff
@@ -41,6 +43,7 @@ public CommandResult execute(Model model) {
             } else {
                 result = "Next lesson:\n" + lesson.toString();
             }
+            new ModuleViewCommand(lesson.getModuleCode()).execute(model);
```

Comment:
I think will have null pointer exception when there is no next class (e.g. when the class list is empty).

Reply:
Yea, I will shift that thing into the else clause soon!